docs/users/user_management.md

```diff
@@ -103,6 +103,21 @@ A password that has been used before can be used again.
 
 This rule is valid by default when password expiration is set.
 
+### Breached passwords
+
+You can set a rule that prevents using passwords which have been exposed in a public breach.
+To do this, in the **User account (ezuser)** Field definition, select "Password must not be contained in a public breach".
+
+![Protection against using breached passwords](password_breached.png)
+
+This rule checks the password against known password dumps by using the https://haveibeenpwned.com/ API.
```

Comment:
```suggestion
This rule checks the password against known password dumps by using the https://haveibeenpwned.com/ API. It does not check passwords that are in use, so it will not block login for anyone. It applies to new passwords when users are changing their passwords. 
```

I realised it might be good to mention this point.

Reply:
@Steveb-p We could check existing passwords during login, but I don't want to do that.

(I was considering to add a "check my password"-button on the login form and/or in the user settings, but concluded that's an unnecessary complication. When enabled it will apply to all new users registering, and if site owners really want to they can enable it and then expire all passwords. That will ensure all passwords are un-breached, and all users are pissed off 😁 )